tests/unit/sagemaker/test_deserializers.py

```diff
@@ -119,3 +120,28 @@ def test_numpy_deserializer_from_npy_object_array(numpy_deserializer):
     result = numpy_deserializer.deserialize(stream, "application/x-npy")
 
     assert np.array_equal(array, result)
+
```

Comment:
(optional/food for thought) we're probably getting close to where it might make more sense to split this file up into multiple test files